packages/jsii/lib/compiler.ts

```diff
@@ -223,8 +223,18 @@ export class Compiler implements Emitter {
         target: COMPILER_OPTIONS.target && ts.ScriptTarget[COMPILER_OPTIONS.target],
         jsx: COMPILER_OPTIONS.jsx && Case.snake(ts.JsxEmit[COMPILER_OPTIONS.jsx]),
       },
-      include: [pi.tsc?.rootDir ? `${pi.tsc.rootDir}/**/*.ts` : '**/*.ts'],
-      exclude: ['node_modules'].concat(pi.excludeTypescript),
+      include: [
+        pi.tsc?.rootDir
+          ? path.join(pi.tsc.rootDir, '**', '*.ts')
+          : path.join('**', '*.ts')
+      ],
+      exclude: [
+        'node_modules',
+        ...pi.excludeTypescript,
+        ...pi.tsc?.outDir != null
```

Comment:
For consistency:

```suggestion
        ...pi.tsc?.outDir
```

Reply:
I'd rather fix the other way around (empty sting is falsey)